src/objc_stubs.c

```diff
@@ -10,11 +10,12 @@
  */
 
 #include "arraytypes.h"
-#include "class.c"
+#include "aggregate.h"
 #include "mars.h"
 #include "objc.h"
```

Comment:
This is why I don't like C/C++. Can we just make the D port default and be done with it :smiley: